execution/gethexec/espresso_sequencer.go

```diff
@@ -117,7 +121,21 @@ func (s *EspressoSequencer) Start(ctxIn context.Context) error {
 
 // Required methods for the TransactionPublisher interface
 func (s *EspressoSequencer) PublishTransaction(parentCtx context.Context, tx *types.Transaction, options *arbitrum_types.ConditionalOptions) error {
-	if err := s.hotShotState.client.SubmitTransaction(parentCtx, tx); err != nil {
+	var txnBytes, err = json.Marshal(tx)
+	if err != nil {
+		return err
+	}
+	//	json.RawMessage is a []byte array, which is marshalled as a base64-encoded string.
+	//	Our sequencer API expects a JSON array.
+	payload := make([]uint8, len(txnBytes))
```

Comment:
@ImJeremyHe are you sure this works correctly and doesn't lead to go using base64 for the bytes?

I ask because when I was debugging the RLP PR

- https://github.com/EspressoSystems/nitro-espresso-integration/pull/24

I think I noticed that uint8 caused go to use base64 that's why I used `uint` instead. I'm not 100% sure anymore though.

Reply:
Done in the client repo now. https://github.com/EspressoSystems/nitro-espresso-integration/pull/31/commits/4756980c6f90053e0d4614e12c37ad02043730d7